resources/js/theme-default.js

```diff
@@ -14,16 +14,21 @@ document.addEventListener('DOMContentLoaded', function() {
 
         throttledUpdateHash(entries[0].target.id);
     }, {
-        rootMargin: '-8% 0px -8% 0px', // shrink the intersection viewport
+        rootMargin: '0px 0px -12% 0px', // shrink the intersection viewport
```

Comment:
I think we might need to use `rem` here in order to make all screens, no matter the size, work nicely. I will take a crack at that